README.md

```diff
@@ -1 +1,4 @@
-# leapp-setup
\ No newline at end of file
+# leapp-setup
+This repo contains setup and installation scripts for Leapp, which is used to manage AWS SSO credentials.
+
+See the Panopedia page on Leapp for more details.
```

Comment:
nitpick (non-blocking): Can we add Panopedia page link here?

Reply:
Yup!  Based on the thread based on what we're OK including, I'm going to include links to Panopedia.